uk.co.compucorp.civicrm.hrcontactactionsmenu/hrcontactactionsmenu.php

```diff
@@ -126,17 +126,8 @@ function hrcontactactionsmenu_civicrm_alterSettingsFolders(&$metaDataFolders = N
  * Implementation of hook_civicrm_pageRun
  */
 function hrcontactactionsmenu_civicrm_pageRun(&$page) {
-  CRM_Core_Region::instance('contact-page-inline-actions')->update('default', array(
-    'disabled' => TRUE,
-  ));
-  $template_path = CRM_Core_Resources::singleton()->getPath('uk.co.compucorp.civicrm.hrcontactactionsmenu','templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl');
-  
-  if ($page instanceof CRM_Contact_Page_View_Summary) {
-    CRM_Core_Region::instance('contact-page-inline-actions')->add(array(
-      'template' => $template_path,
-    ));
-  }
-  
   $extName = 'uk.co.compucorp.civicrm.hrcontactactionsmenu';
   CRM_Core_Resources::singleton()->addStyleFile($extName, 'css/contactactions.css');
+  CRM_Core_Resources::singleton()->addScriptFile($extName, 'js/contactactions.js');
+  
```

Comment:
@bmnepali , I think I left a comment earlier that the code should be wrapped in a condition so that this scripts doesn't get added to every page.

Reply:
Fixed!